env_characterization/Action action
---
std_msgs/Bool result